torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -191,6 +191,12 @@ def write_header(self):
 
     def write_get_raw_stream(self, index, graph=None):
         name = f"stream{index}"
+        if self.device == "xpu":
```

Comment:
`device_codegen` provide `aoti_get_stream` and Intel GPU also overrides this function. Why is the check still required?

Reply:
Hi, @EikanWang  thanks for your reminder, resolved, please have a review.

> `device_codegen` provide `aoti_get_stream` and Intel GPU also overrides this function. Why is the check still required?